src/views/Orders.vue

```diff
@@ -102,6 +102,11 @@
                 <ion-label class="ion-text-wrap">{{ batch?.jobName }}</ion-label>
                 <ion-note slot="end">{{ batch?.runTime ? getTime(batch.runTime) : '' }}</ion-note>
               </ion-item>
+              <ion-item-options side="start">
+                <ion-item-option @click="skipBatch(batch)" color="secondary" style="color:black">
```

Comment:
I think we don't need any style property here.

Reply:
Done sir, removed!